### DISCUSSION TITLE
使用小程序客服消息，一直提示服务故障

### DISCUSSION THREAD

**Discussion by sp5i5s** [w7corp/easywechat#2664] [General]
个人帐号类型的小程序，确认没有使用第三方服务，已经开通并配置了消息推送服务，具体配置如下：

EasyWeChat版本：5
消息加密方式：兼容
数据格式：XML

代码大致如下：
`private function mpInit()
    {
        $config = [
            'app_id' => 'xx',
            'secret' => 'xx',
            'token' => 'xx',
            'response_type' => 'array',
            // 'aes_key' => 'ryBKn8i1O9HZk1dXXfl0Ry1EXKFgD0Poz1lCp1AY36m', 
            'log' => [
                'level' => 'debug',
                'file' => __DIR__ . '/wechat.log',
            ],
        ];
        self::$app = Factory::miniProgram($config);
    }

    public function checkSignature()
    {
        self::$app->server->push(function ($message) {
            return "您好！欢迎使用!";
        });

        $response = self::$app->server->serve();
        // 将响应输出
        $response->send();
        exit;
    }`
如果配置了aes_key，则没有报错，但一直无法回复消息，没有配置的话就直接提示“该小程序提供的服务出现故障，请稍后再试”

**Comment by sp5i5s** [comment]
已经解决，谢谢观注~

**Comment by lanlangdeai** [comment]
请问, 如何解决的呢?
